yarn.lock

```diff
@@ -2572,6 +2572,11 @@
   dependencies:
     "@types/node" ">= 8"
 
+"@ovh-ux/manager-config@^0.4.0":
```

Comment:
Shouldn't be there.

```sh
aleblanc@laptop1264481: ~/Code/github/ovh/manager on master
$ cat packages/manager/modules/config/package.json | json version
1.1.1
```

I will ask a rebase from dependabot.

Reply:
Issue in on `master` branch.

```sh
$ git rev-parse HEAD
ac395ed11305408489acb7bcf0ec27b72bd12543

$ yarn install

$ git status -s
 M yarn.lock

$ git diff
diff --git a/yarn.lock b/yarn.lock
index 4cfb0199570a..dfb7c8492b60 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -2572,6 +2572,11 @@
   dependencies:
     "@types/node" ">= 8"
 
+"@ovh-ux/manager-config@^0.4.0":
+  version "0.4.0"
+  resolved "https://registry.yarnpkg.com/@ovh-ux/manager-config/-/manager-config-0.4.0.tgz#81171e361b8400870821632adddea7b46a240395"
+  integrity sha512-ZiLfkRNwCqnyQvYpN9Frx/WKXE9JEQhtkmoxIdhzDtkBjvuIHbC9uGH7rzh771xfuZcUPv8hmidwf0cjdpXCjQ==
+
 "@ovh-ux/ng-ovh-api-wrappers@^4.0.7":
   version "4.0.7"
   resolved "https://registry.yarnpkg.com/@ovh-ux/ng-ovh-api-wrappers/-/ng-ovh-api-wrappers-4.0.7.tgz#f07cff3c42ef1c9d1655e2300617528a7d8b2f4f"
```

I will open a Pull Request in order to fix the `yarn.lock` file.